_data/people/jpivarski.yml

```
@@ -518,3 +518,37 @@ presentations:
   - servicex
   - uproot
   - awkward
+- title: Loopy and Unloopy programming techniques (tutorial)
+  date: 2022-07-11
+  url: https://github.com/jpivarski-talks/2022-07-11-scipy-loopy-tutorial#readme
+  meeting: SciPy 2022 Conference
+  meetingurl: https://www.scipy2022.scipy.org/
+  location: Austin, Texas
+  focus-area:
+  - as
+  project:
+  - awkward
+- title: Awkward Arrays: Accelerating scientific data analysis on irregularly shaped data (poster)
```

YAML's optional quotation marks can be a sucker-punch sometimes. I'd rather the format just always require them.

```suggestion
- title: "Awkward Arrays: Accelerating scientific data analysis on irregularly shaped data (poster)"
```